Sources/SwiftDocC/SwiftDocCMarkdownOutput/MarkdownOutputNode.swift

```diff
@@ -0,0 +1,199 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2025 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+public import Foundation
+
+// Consumers of `MarkdownOutputNode` in other packages should be able to lift this file and be able to use it standalone, without any dependencies from SwiftDocC.
+
+/// A markdown version of a documentation node.
+@_spi(MarkdownOutput)
+public struct MarkdownOutputNode: Sendable {
+
+    /// The metadata about this node
+    public var metadata: Metadata
+    /// The markdown content of this node
+    public var markdown: String = ""
+    
+    public init(metadata: Metadata, markdown: String) {
+        self.metadata = metadata
+        self.markdown = markdown
+    }
+}
+
+extension MarkdownOutputNode {
+    public struct Metadata: Codable, Sendable {
+    
+        static let version = "0.1.0"
+        
+        public enum DocumentType: String, Codable, Sendable {
+            case article, tutorial, symbol
+        }
+        
+        public struct Availability: Codable, Equatable, Sendable {
+            
+            public let platform: String
+            public let introduced: String?
+            public let deprecated: String?
+            public let unavailable: Bool
+                        
+            public init(platform: String, introduced: String? = nil, deprecated: String? = nil, unavailable: Bool) {
+                self.platform = platform
+                // Can't have deprecated without an introduced
+                self.introduced = introduced ?? deprecated
```

Comment:
I don't know if this is correct. It's possible to specify `@available(macOS, deprecated: 26, message: "This symbol is deprecated")` as an availability attribute in Swift. 

Generally, no introduced version means that a symbol is available as far back as the framework itself supports.